Bottom: c379f670a3ea943e8b7352245547f92a4986bad4
Top:    b40079828860aa9f2c2781ef2fa534dca65ffa47
Author: Jason Wang <jasowang@redhat.com>
Date:   2017-08-31 15:35:38 +0800

Refresh of vhost-introduce-helper-to

---

diff --git a/drivers/vhost/vhost.h b/drivers/vhost/vhost.h
index ec21473..8e968dc 100644
--- a/drivers/vhost/vhost.h
+++ b/drivers/vhost/vhost.h
@@ -230,7 +230,7 @@ ssize_t vhost_chr_write_iter(struct vhost_dev *dev,
 int vhost_init_device_iotlb(struct vhost_dev *d, bool enabled);
 int vhost_prefetch_desc_indices(struct vhost_virtqueue *vq,
                                 struct vring_used_elem *heads,
-				u16 num);
+			        u16 num);
 
 #define vq_err(vq, fmt, ...) do {                                  \
 		pr_debug(pr_fmt(fmt), ##__VA_ARGS__);       \
